Project-Descriptions-and-Plans/CV32E4_TBD/Project_Concept_for_CV32E4_TBD_Nov-27_2023.md

```diff
@@ -0,0 +1,250 @@
+# OpenHW Project Concept Proposal: CV32E4_TBD
+*The PC proposal introduces the project and explains the market drivers and the "why"*
+
+# Title of Project - "CV32E4_TBD"
+# Project Concept Proposal
+## Date of proposal - 2023-11-27
+## Author(s) - Manuel PEZZIN (CEA)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is a continuation of CV32E40Pv2 project.
+Main goal is to support official RVB (and a part of RVK) and (upcoming) RVP
+extensions while keeping backward compatibility with CV32E40Pv1/v2 (eg XPULP
+custom extensions are kept and even extended).
+Additionnal features may also be considered, like:
+* CV-X-IF support
+* CLIC support
+* Improved OBI interface support (bus error in particular)
+
+This new project will keep functionnal backward compatibility with CV32E40Pv2,
+but with relaxed rules compared to previous project. In particular, formal equivalence
+with previous design will not be required to make design work easier.
```

Comment:
but how about v1?

Reply:
As v2 is already backward-compatible with v1, I consider this will be (implicitly) the case, provided v1 configuration (no XPULP, no RVB, no RVP, etc.) remains in verification plan.

Reply:
ok, thx - I think we can add 1 line to say that explicitly :)